server_versions.json

```diff
@@ -47,31 +47,27 @@
     "name": "Paper",
     "type": "SERVER",
     "isPaperclip": true,
-    "latestVersion": "1.20.5",
+    "latestVersion": "1.20.6",
     "downloadLinks": [
       {
-        "version": "1.20.5",
-        "link": "https://api.papermc.io/v2/projects/paper/versions/1.20.5/builds/6/downloads/paper-1.20.5-6.jar"
+        "version": "1.20.6",
+        "link": "https://api.papermc.io/v2/projects/paper/versions/1.20.6/builds/51/downloads/paper-1.20.6-51.jar"
       },
       {
         "version": "1.20.4",
-        "link": "https://api.papermc.io/v2/projects/paper/versions/1.20.4/builds/435/downloads/paper-1.20.4-435.jar"
+        "link": "https://api.papermc.io/v2/projects/paper/versions/1.20.4/builds/496/downloads/paper-1.20.4-496.jar"
       },
       {
         "version": "1.20.2",
-        "link": "https://api.papermc.io/v2/projects/paper/versions/1.20.2/builds/291/downloads/paper-1.20.2-291.jar"
+        "link": "https://api.papermc.io/v2/projects/paper/versions/1.20.2/builds/318/downloads/paper-1.20.2-318.jar"
       },
       {
         "version": "1.20.1",
         "link": "https://api.papermc.io/v2/projects/paper/versions/1.20.1/builds/196/downloads/paper-1.20.1-196.jar"
       },
-      {
-        "version": "1.20",
```

Comment:
Why did you remove this? I would leave it there

Reply:
Yeah, that's true, but there are people using that version and because we're going to use this file also for the v2 this could break the cloud for some people, so we have to leave it there.

That's also applying for all other versions, so I'm not going to reply to every comment